ui/components/app/detected-token/detected-token-ignored-popover/index.scss

```diff
@@ -0,0 +1,13 @@
+.detected-token-ignored-popover {
+  &__ignore-button {
+    margin-right: 8px;
```

Comment:
Does this popover look as it should in RTL?  Just wondering because of the left/right properties.

Reply:
updated